dts/common/nordic/nrf54h20_enga.dtsi

```diff
@@ -168,6 +168,11 @@
 				interrupts = <64 NRF_DEFAULT_IRQ_PRIORITY>,
 					     <65 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			cpuapp_ieee802154: ieee802154 {
+				compatible = "nordic,nrf-ieee802154";
+				status = "disabled";
+			};
```

Comment:
shouldn't this be a child of the radio node, as do other nordic socs?

Reply:
The radiocore equivalent of this node is placed under radio. The app one is placed differently on nRF5340 as well.

Reply:
> The app one is placed differently on nRF5340 as well.

Exactly, we solved this issue [the same way for nRF5340](https://github.com/zephyrproject-rtos/zephyr/pull/51123#discussion_r991167424). Adding an actual radio node to the application core would be misleading and would lead to errors should someone actually try to use it.